components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -0,0 +1,51 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Json = require('Module:Json')
+local Table = require('Module:Table')
+
+local AutomaticPointsTable = Class.new(
+	function(self, frame)
+		self.frame = frame
+		self.args = Arguments.getArgs(frame)
+		self:parseInput()

```

Comment:
Doesn't exist. Also:

```suggestion
		self:parseInput(self.args)

```

Reply:
I've already attached the args to the instance of the AutomaticPointsTable, and I'm using it in all the function as **self.args**, do I need to pass it around instead?

Reply:
Yes, is a much better pattern. Makes the functions more independent and easier to reason about.